eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -65,48 +66,56 @@ jobs:
           TestTargetFramework: netcoreapp2.1
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
           ArmTemplateParameters: $(azureCloudArmParameters)
+          Location: "westus"
         Windows_NetFramework (AzureCloud):
```

Comment:
This matrix looks so old and out dated... where is our fancy json matrix file :)

Reply:
I started down that path, but I'll need to add a `dependsOn` parameter pass-through to the underlying job, so that it can reference the job variables outputted by the smoke test eligibility job. I still plan to do it, but it will take a tools change first, so I wanted to get this up as an immediate solution without blocking it on that change.